Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/TextToolbar.cs

```diff
@@ -36,9 +36,9 @@ public partial class TextToolbar : Control
         internal const string HeadersElement = "Headers";
 
         /// <summary>
-        /// Gets access to Generic Buttons that activate Formatter Methods
+        /// Gets access to Generic Buttons to instantiate for Creating Formatters (does not get the Current Instance from Toolbar)
         /// </summary>
-        internal CommonButtons CommonButtons { get; }
+        public CommonButtons CommonButtons { get; }
```

Comment:
Since these are 'standard' buttons, essentially created from a factory class; I think this should be static.  Then you wouldn't need as verbose a comment.

Reply:
I could have it Instantiated as Protected in the Formatter Base Class, or on demand, what do you think?

Reply:
Scratch that, I'm making it On Demand, so it simplified GC for Formatters that might have no Common Buttons.